dashboard/views_test.py

```diff
@@ -175,6 +177,22 @@ def test_with_runs(self, mocked_refresh):
                     if course_data['runs'][0]['course_id'] == "course-v1:MITx+8.MechCX+2014_T1":
                         assert course_data['runs'][0]['status'] == CourseStatus.UPGRADE
 
+    @patch('backends.utils.refresh_user_token', autospec=True)
+    def test_certificates_enrollments(self, mocked_refresh):
+        """
+        Make sure we call the proper api functions
```

Comment:
the reason is because we test how the called functions work separately. So assuming they work properly the only thing to test here is that they are called in the right way.